src/plugins/kibana/public/visualize/editor/editor.js

```diff
@@ -116,6 +116,7 @@ uiModules
     };
 
     $state = new AppState(stateDefaults);
+    $scope.uiState = $state.makeStateful('uiState');
```

Comment:
This line is not necessary.
https://github.com/elastic/kibana/pull/5953/files#diff-eaa764287bd619662df5d1a5d17486a1R142